Open Judge System/Web/OJS.Web/Infrastructure/Filters/ClearContestFromProblemFilter.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace OJS.Web.Infrastructure.Filters
+{
+    using System;
+    using System.Web.Mvc;
+    using OJS.Common.Constants;
+    using OJS.Services.Cache;
+    using OJS.Web.Areas.Administration.ViewModels.Contest;
+    using OJS.Web.Infrastructure.Filters.Attributes;
+    using OJS.Web.Infrastructure.Filters.Contracts;
+
+    public class ClearContestFromProblemFilter : IActionFilter<ClearContestFromProblemAttribute>
```

Comment:
Add documentation explaining what is the purpose of this filter and what it does.

Reply:
Done